src/test/bin/quic_gtest.cpp

```diff
@@ -1769,7 +1770,8 @@ TEST_P(WithSendArgs1, Send) {
             GetParam().UseSendBuffer,
             GetParam().UnidirectionalStreams,
             GetParam().ServerInitiatedStreams,
-            false); // FifoScheduling
+            false,  // FifoScheduling
+            true); // SendUdpToQtipListener
```

Comment:
Instead of changing the behavior of an existing test, let's add a new one, specific to QTIP. I don't want to lose any coverage.

Reply:
Fixed with: https://github.com/microsoft/msquic/pull/4803/commits/6b4394a4ede8bb6d33bdb52a51b910134c0e559a